vars.tf

```diff
@@ -0,0 +1,106 @@
+variable "region" {
+  default = "us-west2"
+}
+
+variable "availability_zone" {
+  description = "The zone where the cluster will be deployed [a,b,...]"
+  default     = ["a", "b"]
+  type        = list(string)
+}
+
+variable "instance_group_name" {
+  description = "The name of the GCP instance group"
+  default     = "redpanda-group"
+}
+
+variable "broker_count" {
+  description = "The number of Redpanda brokers to deploy."
+  type        = number
+  default     = "3"
+}
+
+variable "ha" {
+  description = "Whether to use placement groups to create an HA topology"
```

Comment:
[question] This is necessary for Rack Awareness, right? Should we explicitly say this here?

Reply:
added to comment in variable